app/Http/Controllers/AdminController.php

```diff
@@ -228,50 +232,98 @@ public function updateTags (Request $request)
+        $photo = $this->filterPhotos()
+            ->when($request->skip, function ($q) use ($request) {
+                $q->skip((int) $request->skip);
+            })
+            ->where('verification', 0.1)
+            ->first();
 
         // Load the tags for this photo if it exists
         if ($photo)
         {
             $photo->tags();
         }
 
-        // Count photos that have been uploaded, but not tagged or submitted for verification
-        $photosNotProcessed = Photo::where([
-                ['verification', 0],
-                ['user_id', '!=', 3233],
-                ['user_id', '!=', 5292]
-        ])->count();
-
-        // Count photos submitted for verification
-        $photosAwaitingVerification = Photo::where([
-            ['verified', '<', 2], // not verified
-            ['verification', '>', 0], // submitted for verification
-            ['user_id', '!=', 3233],
-            ['user_id', '!=', 5292]
-        ])->count();
-
         if (!$photo)
         {
-            $photo = Photo::where([
-                'verification' => 0,
-                ['user_id', '!=', 3233], // dont load freds data
-                ['user_id', '!=', 5292]  // or sarahs
-            ])->first();
+            // Photos that have been uploaded, but not tagged or submitted for verification
+            /** @var Photo $photo */
+            $photo = $this->filterPhotos()
+                ->when($request->skip, function ($q) use ($request) {
+                    $q->skip($request->skip);
+                })
+                ->where('verification', 0)
+                ->first();
         }
 
+        // Count photos that are uploaded but not tagged
+        $photosNotProcessed = $this->filterPhotos()
+            ->where('verification', 0)
+            ->count();
+
+        // Count photos submitted for verification
+        $photosAwaitingVerification = $this->filterPhotos()
+            ->where([
+                ['verified', '<', 2], // not verified
+                ['verification', '>', 0], // submitted for verification
+            ])
+            ->count();
+
         return [
             'photo' => $photo,
             'photosNotProcessed' => $photosNotProcessed,
             'photosAwaitingVerification' => $photosAwaitingVerification
         ];
     }
+
+    /**
+     * Returns all the countries that have unverified photos
+     * and their totals
+     */
+    public function getCountriesWithPhotos(): Collection
+    {
+        $totalsQuery = Photo::query()
+            ->selectRaw('country_id, count(*) as total')
+            ->whereIn('verification', [0, 0.1])
+            ->whereNotIn('user_id', $this->usersToSkipVerification())
+            ->groupBy('country_id');
+
+        // Using DB to avoid extra appended properties
+        return DB::table('countries')
+            ->selectRaw('id, country, q.total')
+            ->rightJoinSub($totalsQuery, 'q', 'countries.id', '=', 'q.country_id')
```

Comment:
what is `q` ? query I guess..

Reply:
Yes, every subquery needs an alias, 'q' is quite common.